packages/editor/src/client/routes/articleList.tsx

```diff
@@ -122,6 +133,9 @@ export function ArticleList() {
           data={articles}
           sortColumn={sortField}
           sortType={sortOrder}
+          rowClassName={rowData => {
+            return rowData?.id === highlightedRowId ? 'highlighted-row' : ''
```

Comment:
It's different code styles but can we remove the return keyword and parentheses? What do you think?

Reply:
Yes, that will work too 👍